functionary/core/models/environment.py

```diff
@@ -0,0 +1,58 @@
+""" Environment model """
+import uuid
+from typing import TYPE_CHECKING
+
+from django.db import models
+
+if TYPE_CHECKING:
+    from core.models import Team
```

Comment:
May want to add a note that this is done to avoid a circular dependency for type checking. If not here, somewhere else that would be prominent.

Reply:
Comment added.